healthcare/api-client/hl7v2/hl7v2_messages_test.py

```diff
@@ -28,8 +29,14 @@
 project_id = os.environ['GOOGLE_CLOUD_PROJECT']
 service_account_json = os.environ['GOOGLE_APPLICATION_CREDENTIALS']
 
-dataset_id = 'test_dataset_{}'.format(int(time.time()))
-hl7v2_store_id = 'test_hl7v2_store-{}'.format(int(time.time()))
+# assume we can access the OS provied raondom source.
```

Comment:
Other samples generally use [uuid](https://docs.python.org/3/library/uuid.html) module to generate a random string for resource names. 

https://github.com/GoogleCloudPlatform/python-docs-samples/blob/fe705fc42371d7f9697b2ef3192bc59008d7b136/dialogflow/cloud-client/create_document_test.py#L18-L28

Reply:
Certainly we can use uuid, but having the time portion in the string has a large benefit, especially for resources which can leak. We can safely delete old leaked resources manually, or even automatically.

Leaked resources are potentially sources of test flakiness, so I'd stick to having the time portion.

Thoughts?

Reply:
TIL uuid has time field, so yeah uuid is fine